examples/BarChart.elm

```diff
@@ -0,0 +1,27 @@
+module OrdinalBands where
+
+import Svg exposing (Svg)
+import Plot exposing (..)
+import Plot.Scale as Scale
+import Plot.Axis as Axis
+
+main : Svg
+main =
+  let
+    xScale = Scale.ordinalBands ["a", "b", "c", "d"] (0, 800) 0.2 0.2
+    yScale = Scale.linear (0, 400) (800, 0) 10
```

Comment:
To me, it depends on how you think about it. From drawing a graphics perspective, I agree that (0, 0) being top left makes sense. When working with data plots the origin (0, 0) is the bottom left. When using a plotting library I would expect that I should be able to directly pass my data in and not have to worry about flipping the y-axis so that it displays correctly.

I think maintaining consistency with other libraries is great if the intention of this library is to be more graphics oriented than plot oriented. Since we only plan on using it for plots then we can always wrap this up in a way that we never have to worry about remembering to flip the y-axis.